docs/contracts.rst

```diff
@@ -484,42 +439,6 @@ and the arguments are ambiguous.
         1
 
 
-
-.. _event-log-object:
```

Comment:
Slug is referenced elsewhere in the docs (via `:ref:`), which produces a warning.

Section below also referenced.